pmm-app/tests/pages/dashboardPage.js

```diff
@@ -320,6 +320,7 @@ module.exports = {
       "//span[contains(text(),'No Data')]//ancestor::div[contains(@class,'panel-container')]//span[contains(@class,'panel-title-text')]",
     collapsedDashboardRow: "//div[@class='dashboard-row dashboard-row--collapsed']/a",
     annotationMarker: "(//div[contains(@class,'events_marker')])",
+    clearSelection: "//a[@ng-click='vm.clearSelections()']",
```

Comment:
Pls don't hook to such unreliable selector - they are brittle, will break soon. There should be a better way